apis/kms/v1alpha1/custom_types.go

```diff
@@ -0,0 +1,10 @@
+package v1alpha1
+
+// CustomKeyParameters are custom parameters for Key.
+type CustomKeyParameters struct {
+	// Specifies whether the CMK is enabled.
+	Enabled *bool `json:"enabled,omitempty"`
+
+	// Specifies how many days the Key is retained when scheduled for deletion. Defaults to 30 days.
+	PendingWindowInDays *int `json:"pendingWindowInDays,omitempty"`
```

Comment:
```suggestion
	PendingWindowInDays *int64 `json:"pendingWindowInDays,omitempty"`
```

Just to be consistent with others.